cogs/guess.py

```diff
@@ -0,0 +1,93 @@
+"""importsss (that i don't know if it belongs here)
+
+    you can get rid of excess comments"""
+import time
+import random
+from discord.ext import commands
+from random import choice
+from collections import Counter, defaultdict, namedtuple
+import discord
+import os
+import asyncio
+import re
+"""Guessing game """
+class Guess:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(name="guess")
+    async def mycom(self):
+        # welcoming the user
+        await self.bot.reply("guess what word I'm thinking of!")
+
+        # thinking
+        await asyncio.sleep(3)
+
+        # start guess
+        await self.bot.say("Start guessing...(hint: no caps)")
+        await asyncio.sleep(0.5)
+
+        # make secret
+        secret = "its gonna change"
+
+        # Probably need to create a word list and grab words from there
+        items = ["here", "are", "some", "strings", "of","which", "we", "will", 
+         "select", "one", "two", "three", "four", "five", "six", 
+         "seven", "eight", "nine", "ten", "python", "ruby", "java", "javascript", "traceback", "print", 
+         "random", "input", "raw", "print", "sleep", "time", "secret", "while", "then", "if", 
+         "loop", "in", "else", "break", "empty", "mocha", "matcha", "money", "spam","jazz","jam","spaz", 
+         "binder", "jokes", "chemistry", "person", "matcha", "night", "everyone", "pasta", "craving", 
+         "jolly", "ranch", "lit", "still", "first", "second", "third", "skull", "ketchup", 
+         "ice", "red", "life", "death", "magic", "comments", "class", "secret", "value", 
+         "variable", "guess", "incorrect", "russian", "roulette", "bot", "turn", 
+         "failed", "wood", "stone", "cobblestone", "blanket", "ore", "chest", "poster"]
+        #picks one word from list
+        secret = random.choice(items)
+
+        #creates a variable with an empty value
+        guesses = ''
+        turns = 5
+        while turns > 0:
+            failed = 0
+            for char in secret:
+                if char in guesses:
+                    await self.bot.say(char),
+                else:
+                    await self.bot.say("_"),
+                    failed += 1
+                #if failed is equal to zero
+                # Win text
+            if failed == 0:
+                await self.bot.say("You won! Good job! :aquaThumbsUp:")  
+                # exit the script"""
+            
+                break
+            # ask the user go guess a character"""
+            await self.bot.say("lower case letters please~~!")
+            guess = self.bot.discord.wait_for_message(timeout=None, author=None, channel=None, content=None, check=None)
```

Comment:
Didn't see this before, but you don't need to specify `None` for these keyword arguments since they are optional arguments that default to `None` anyways.

Nonetheless, `channel` must be `context.message.channel`.  You need to get `context` by also adding the following keyword argument to `@commands.command` at the beginning of your method: `pass_context=True`.

Please also set a timeout, and handle the case if you don't get a message back in time (e.g. `guess is None`).

Reply:
In addition, you'll need to add another positional argument to your method, which will be used for context.  See some of the other cogs if this is not clear.